service/cloudconfig/v_0_1_0_master_template.go

```diff
@@ -38,9 +45,19 @@ func v_0_1_0MasterTemplate(customObject kvmtpr.CustomObject, certs certificatetp
 
 type v_0_1_0MasterExtension struct {
 	certs certificatetpr.AssetsBundle
+	keys  []byte
```

Comment:
This looks like `keys` is now only one key. IMO it should be what we get just like with the certs.

Reply:
you mean something like `encryptionKey` instead

Reply:
I mean there is a `certs` argument given which has the type `certificatetpr.AssetsBundle`. This is applied to the `v_0_1_0MasterExtension` type as it is. For the key you do it differently and I wonder why. When type A is given we should probably just use type A and not type B.

Reply:
Ok got you. So it would be preferable to validate the map, but retrieving the key in `.Files()` when creating `EncryptionConfig`

I'll do that

Reply:
Yeah. Cool.

Reply:
ok done